motr/ut/idx_dix.c

```diff
@@ -786,7 +786,7 @@ struct m0_client* st_get_instance()
 #include "dtm0/service.h"
 
 #define M0_FID(c_, k_)  { .f_container = c_, .f_key = k_ }
-static void st_one_dtm0_op_idx_create(void)
```

Comment:
I don't see st_one_dtm0_op_idx_create being used anywhere. If the only reason for making this non-static is to overcome compiler error then consider marking it M0_UNUSED instead.